Installation_Guide/Installation-Guide.rst

```diff
@@ -119,7 +110,7 @@ Supported Operating Systems
 Ubuntu
 =========
 
-**Note**: AMD ROCm only supports Long Term Support (LTS) versions of Ubuntu. Versions other than LTS may work with ROCm, however, they are not officially supported. 
+**Note**: AMD ROCm only supports Long Term Support (LTS) versions of Ubuntu. Versions other than LTS may work with ROCm, however, they are not officially supported. Debian-based distributions other than Ubuntu LTS are not officially supported, they might however work. In case of issues or bugs, retest on Ubuntu LTS, if possible.
```

Comment:
How are you inferring this?

Reply:
There are many people using non-LTS or non Ubuntu distros, like Debian (including me), with a success and without major issues.